### PR TITLE
Emails

### PR DESCRIPTION
**Pull request by saoudnazir**
Please add a short description of your Pull Request here.

{ https://trello.com/c/JufwKhPk }
{ Link to applicable Invision designs }

### Done Checklist

* [x] Task has been completed as per the Trello card
* [ ] All acceptance criteria are met
* [x ] Developer has tested this locally
* [x] Developer has reviewed their own PR
* [ ] Design has been reviewed by design team
* [ ] Tests have been added or updated
* [ ] Views are responsive for mobile devices
* [ ] Feature files have been written or updated
* [ ] Changelog updated

### CODE REVIEW

File: meteor-app/imports/startup/server/email.js [hunk 1]
```diff
@@ -33,5 +37,96 @@ Meteor.methods({
...
+      if (member.sessions.length >= 5) {
+        Meteor.call(
+          'sendPassEmail',
+          member.email,
+          member.name,
+          member.sessions.length,
+          Meteor.settings.private.validPassID
+        )
+
+        debug('Sending Email to ' + member.name)
```
Comment (mikkelking) [inline meteor-app/imports/startup/server/email.js]: It's worth being more descriptive of which email you are sending here, eg 'sending passes remaining email'...

File: meteor-app/imports/startup/server/email.js [hunk 2]
```diff
@@ -33,5 +37,96 @@ Meteor.methods({
...
+      } else if (member.sessions.length >= 10) {
+        Meteor.call(
+          'sendPassEmail',
+          member.email,
+          member.name,
+          member.sessions.length,
+          Meteor.settings.private.expiredPassID
+        )
+        debug('Sending Email to ' + member.name)
```
Comment (mikkelking) [inline meteor-app/imports/startup/server/email.js]: As above, describe which email you are sending here.

File: meteor-app/imports/startup/server/email.js [hunk 3]
```diff
@@ -33,5 +37,96 @@ Meteor.methods({
...
+          Meteor.settings.private.expiredMembershipID
+        )
+      })
+    })
+  },
+  sendMembershipReminderEmail() {
+    Members.find({}).forEach(member => {
+      Purchases.find({
+        memberId: member._id,
+        $or: [{ code: 'PA-MEMB-12' }, { code: 'PA-MEMB-3' }],
```
Comment (mikkelking) [inline meteor-app/imports/startup/server/email.js]: Shouldn't this use the regular expression as above?

Comment (saoudnazir) [reply meteor-app/imports/startup/server/email.js]: Actually I did but i think it has changed when I pulled 'shop' branch. I will change it again.

**Comment by mikkelking** [comment]
Merging this PR anyway, even though it's not really complete. There is more work to do, so let's just create another branch off develop and do the last bit properly

**Pull request merged**